lib/src/base-styles/_nx-card.scss

```diff
@@ -0,0 +1,173 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-card-container {
+  display: flex;
+  margin: $nx-spacing-md 0 $nx-spacing-l 0;
+}
+
+.nx-card {
+  background-color: #fff;
+  border-radius: $nx-border-radius;
+  box-sizing: border-box;
+  padding: $nx-spacing-l $nx-spacing-xxl;
+}
+
+.nx-card__call-out {
+  .nx-icon {
+    font-size: 48px;
+  }
+
+  .nx-icon--large {
+    font-size: 64px;
+  }
+}
+
+.nx-call__call-out-text {
+  @include bold();
+
+  font-size: 32px;
+}
+
+// Vertically oriented cards appear in a row container
+.nx-card-container--row {
+  @include container-horizontal;
+
+  .nx-card {
+    display: flex;
+    flex-direction: column;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    margin: 0 $nx-spacing-md 0 0;
+    text-align: center;
+    min-width: 230px;
+    max-width: 330px;
```

Comment:
Interesting behavior with the dynamic width.  When using small words, I can't get the cards to expand beyond 286.17px wide.  It appears that they prefer to expand beyond min-width rather than wrapping text lines, up to that point.  After that they start wrapping the text and don't expand their overall width any further.  The only way I can get them wider is to include an individual word which is very long - and even then, it doesn't start to increase the width until well _after_ the word is already too wide for the container.

Reply:
> Interesting behavior with the dynamic width. When using small words, I can't get the cards to expand beyond 286.17px wide. It appears that they prefer to expand beyond min-width rather than wrapping text lines, up to that point. After that they start wrapping the text and don't expand their overall width any further. The only way I can get them wider is to include an individual word which is very long - and even then, it doesn't start to increase the width until well after the word is already too wide for the container.

Coming back to this today, I see that the card in question, with the hard-coded example content (and not my custom content from messing with it) is now 301px wide :man_shrugging: